src/block/SmithingTable.php

```diff
@@ -31,10 +31,7 @@
 final class SmithingTable extends Opaque{
 
 	public function onInteract(Item $item, int $face, Vector3 $clickVector, ?Player $player = null, array &$returnedItems = []) : bool{
-		if($player !== null){
-			$player->setCurrentWindow(new SmithingTableInventory($this->position));
-		}
-
+		$player?->setCurrentWindow(new SmithingTableInventory($this->position));
```

Comment:
Unrelated change

Reply:
is this bad though?

Reply:
It's unrelated to the PR objective, so it shouldn't be changed.